src/traits/drop.md

```diff
@@ -28,3 +28,12 @@ fn main() {
     println!("Exiting main");
 }
 ```
+
+<details>
+
+* `drop` is called automatically, but it can be called manually like in this example.
```

Comment:
That's not really true, `drop` doesn't do anything special beyond what any other function that the value is moved into does. It is literally an empty function, it's just useful to signal intent sometimes.

Reply:
You are right, it is not special and you could implement your own empty `drop2<T> (t: T) {}` and it will work the same. I guess what makes `drop` different is that it is called when an owned object goes out of scope. Calling it manually makes it execute early, and it makes it that the compiler won't call it twice. That's what I meant. 

I am happy with any rewording or if we remove this sentence if it is confusing.